components/stack-entry.js

```diff
@@ -37,6 +37,9 @@ class StackEntry extends PureLayoutComponent {
     if (landscape) {
       illustrationStyle.paddingTop = max(0, round((codeHeight - illustrationHeight) / 2));
       codeStyle.paddingTop = max(0, round((illustrationHeight - codeHeight) / 2));
+
+      illustrationStyle.display = 'table-cell';
+      codeStyle.display = 'table-cell';
```

Comment:
Fixed in https://github.com/skidding/illustrated-algorithms/pull/10/commits/363bbc89a9e5a59781776591b98e5671cfcbf25b